tests/bot_dialogue.rs

```diff
@@ -0,0 +1,62 @@
+#[cfg(feature = "macros")]
```

Comment:
Why this file is called `bot_dialogue`? Seems like it tests macros

Reply:
Renamed to `dialogue_state.rs`.